samples/nrf9160/modem_shell/esp_8266_nrf9160ns.overlay

```diff
@@ -21,20 +21,13 @@
   * 'west build -p -b nrf9160dk_nrf9160_ns -- -DDTC_OVERLAY_FILE=esp_8266_nrf9160ns.overlay -DOVERLAY_CONFIG="overlay-esp-wifi.conf"'
   */
 
-&uart1 {
-	status = "okay";
-	current-speed = <115200>;
-	rts-pin = <0xffffffff>;
-	cts-pin = <0xffffffff>;
-};
-
 &uart3 {
 	status = "okay";
 	current-speed = <115200>;
-	tx-pin = < 0xe >;
-	rx-pin = < 0xf >;
-	rts-pin = <0xffffffff>;
-	cts-pin = <0xffffffff>;
+	tx-pin = <18>;
+	rx-pin = <19>;
```

Comment:
P0.18 and P0.19 are connected to the board controller on the DK. From what I can see, P0.10 and P0.16 are unused. Or you could repurpose some of the LEDs/switches if they're not in use: https://infocenter.nordicsemi.com/topic/ug_nrf91_dk/UG/nrf91_DK/board_controller.html

Reply:
Thanks! Moved to P0.10/16